### ISSUE TITLE
Reproducing performance benchmarks in elfmalloc-performance.md

### ISSUE THREAD

**Issue by mjansson** [ezrosent/allocators-rs#193]
I'm the author of rpmalloc and I was looking at the elfmalloc-performance.md document. However, it doesn't go into details on how you measure the peak memory usage. I would like to reproduce some of your results to see why rpmalloc shows those excessive memory usage numbers seen in your graphs.

Could you expand the doc a bit with more details on how the memory usage is measured?

**Comment by ezrosent** [comment]
My recollection is that some benchmark scripts used ps to get the RSS for a particular process (e.g. https://github.com/cksystemsgroup/scalloc-artifact/blob/master/run/shbench-single.sh#L32)

But others did not (and for whatever reason, it was difficult to adapt them to that purpose). For the latter, I simply eye-balled peak heap size based on task manager, taking an average over the runs that I observed.

Needless to say, if you have trouble reproducing these numbers for rpmalloc I'm happy to remove them, or replace the graph with better numbers. It is quite possible that these memory blowups were due to some aspect of Windows Subsystem for Linux memory management at the time, or some other peculiarity of my setup.
